src/auth/strategies/google.strategy.ts

```diff
@@ -0,0 +1,36 @@
+import { PassportStrategy } from "@nestjs/passport";
+import { Strategy, VerifyCallback } from "passport-google-oauth20";
+import { config } from "dotenv";
+
+import { Injectable } from "@nestjs/common";
+
+config();
+
+@Injectable()
+export class GoogleStrategy extends PassportStrategy(Strategy, "google") {
+  constructor() {
```

Comment:
Inject our own `configService` after you update it to include the 2 env variables you added

```suggestion
  constructor(readonly configService: ConfigService) {
```

Reply:
Will do